src/emulator/functionsEmulator.ts

```diff
@@ -795,6 +861,39 @@ export class FunctionsEmulator implements EmulatorInstance {
     return true;
   }
 
+  addBlockingTrigger(url: string, blockingTrigger: BlockingTrigger): boolean {
+    logger.debug(`addBlockingTrigger`, JSON.stringify({ blockingTrigger }));
+
+    const eventType = blockingTrigger.eventType;
+    if (AUTH_BLOCKING_EVENTS.includes(eventType as any)) {
+      const accessToken = blockingTrigger.options!.accessToken as boolean;
+      const idToken = blockingTrigger.options!.idToken as boolean;
+      const refreshToken = blockingTrigger.options!.refreshToken as boolean;
```

Comment:
nit* I generally like to keep variable declaration close to its use - so move this line to below or pull up `this.blockingFunctionsConfig.forwardInboundCredentials`?

Reply:
Ahh, I had them as separate vars from a previous iteration where I was combining them in this function, fixed it